diff --git a/qwtconfig.pri b/qwtconfig.pri
index 8c5bae5f..73681e50 100644
--- a/qwtconfig.pri
+++ b/qwtconfig.pri
@@ -16,10 +16,10 @@ QWT_VERSION      = $${QWT_VER_MAJ}.$${QWT_VER_MIN}.$${QWT_VER_PAT}
 # Install paths
 ######################################################################
 
-QWT_INSTALL_PREFIX = $$[QT_INSTALL_PREFIX]
+QWT_INSTALL_PREFIX = /usr/local
 
 unix {
-    QWT_INSTALL_PREFIX    = /usr/local/qwt-$$QWT_VERSION-svn
+    QWT_INSTALL_PREFIX    = /usr/local
     # QWT_INSTALL_PREFIX = /usr/local/qwt-$$QWT_VERSION-svn-qt-$$QT_VERSION
 }
 
